### PR TITLE
Escape the plot commands for `plotKicadGerbers` & `plotKicadLayoutSvg`

### LINKED ISSUE #435: `aux-ps-cs` processing fails
https://github.com/eez-open/modular-psu

```
kitspace-processor-1  | failed tester/modular-psu/f60f9882695e93c95d1a24e5c717b5aefe3b9126/mcu-cs/images/layout.svg Command failed: /app/dist/src/tasks/processKicadPCB/plot_kicad_pcb svg /data/checkout/tes/modular-psu/previous designs/mcu/KiCad/EEZ DIB MCU r1B1.kicad_pcb /tmp/kitspace/data/files/tes/modular-psu/f60f9882695e93c95d1a24e5c717b5aefe3b9126/mcu-cs/svg /data/files/tester/modular-psu/f60f9882695e93c95d1a24e5c717b5aefe3b9126/mcu-cs/images/layout.svg
kitspace-processor-1  | Traceback (most recent call last):
kitspace-processor-1  |   File "/app/dist/src/tasks/processKicadPCB/plot_kicad_pcb", line 34, in <module>
kitspace-processor-1  |     board = pcbnew.LoadBoard(input_file)
kitspace-processor-1  |   File "/usr/lib/python3/dist-packages/pcbnew.py", line 9110, in LoadBoard
kitspace-processor-1  |     return _pcbnew.LoadBoard(*args)
kitspace-processor-1  | OSError: Unable to open /data/checkout/tester/modular-psu/previous for reading.
```

### PR DESCRIPTION
**Pull request by AbdulrhmnGhanem**
- Fixes #435

### CODE REVIEW

**Comment by kasbah** [comment]
I'm a bit confused about the shell escape approach actually. (I know it's been added already, but let's discuss it.) Couldn't we put everything into single quotes and then escape any single quotes in the content? The shell escape library seems to be adding an extra layer that makes it less clear what is being "escaped".

File: processor/test/integration/test_projects.ts [hunk 1]
```diff
@@ -294,6 +294,79 @@ describe('projects API', function () {
...
+      assert(
+        r.req.path.includes(hash),
+        `expected '${r.req.path}' to include '${hash}'`,
+      )
+
+      // uppercase user and project name shouldn't matter
+      r = await this.supertest
+        .get(`/files/KITSPACE-TEST-REPOS/spaces-in-kitspace-data-paths/HEAD/${f}`)
+        .redirects()
+      assert(r.status === 200, `expected 200 but got ${r.status} for ${f}`)
```
Comment (kasbah) [inline processor/test/integration/test_projects.ts:366]: Try and make the test as specific to what you are trying to test for as possible.

**Comment by kasbah** [comment]
Sorry, just looked at shell-escape readme, it is what it's doing. I think that's fine.

**Comment by kasbah** [comment]
Could you go through and make sure shell-escape is being used everywhere where we shell out?

File: processor/test/integration/test_projects.ts (hunk 1 above)
Comment (AbdulrhmnGhanem) [reply processor/test/integration/test_projects.ts]: Resolved in fbc4d8370c3d9699fc4e6283ea8256b192061f15.

**Comment by AbdulrhmnGhanem** [comment]
> Could you go through and make sure shell-escape is being used everywhere where we shell out?

Resolved in 8ac91ab.

**Comment by kasbah** [comment]
Can we rename `execEscaped` to `exec` and stop exporting `exec`?

**Comment by kasbah** [comment]
> Can we rename execEscaped to exec and stop exporting exec?

Let's hold off on that since I can see it will cause a lot of merge conflicts with the s3 work I am currently doing.

**Pull request merged**